test/wdio/stencil.config.ts

```diff
@@ -0,0 +1,22 @@
+import { sass } from '@stencil/sass';
+
+import type { Config } from '../../internal/index.js';
+
+export const config: Config = {
+  namespace: 'TestApp',
+  tsconfig: 'tsconfig-stencil.json',
+  outputTargets: [
+    {
+      type: 'dist',
+    },
+  ],
+  plugins: [sass()],
+  buildDist: true,
```

Comment:
Comparing this to `karma/stencil.config.ts`, it looks like we're not bringing `buildEs5` over to this config. I'm OK with that, but wanted to make sure I understand what the motivation for that was (I assume it's speed/warning related, but wanted to make sure I'm not missing anything).

Reply:
It seemed to me that this option generates more output that is not relevant to these tests so I left it out. Frankly, I'm not entirely certain about the specific differences it entails.